core/frontend/src/IModelConnection.ts

```diff
@@ -612,6 +617,23 @@ export abstract class IModelConnection extends IModel {
     }
     return ("number" === typeof this._projectCenterAltitude) ? this._projectCenterAltitude : undefined;
   }
+
+  /**
+   * Gets the context that allows accessing the metadata (ecschema-metadata package) of this iModel.
+   * The context is created lazily when this property is accessed for the first time, with an [[ECSchemaRpcLocater]] registered.
```

Comment:
Should it also mention the necessity for backend to have `ECSchemaRpcImpl.register()` call? Or maybe that should now be done automatically by `IModelHost`?

Reply:
I have limited knowledge of the backend side, I'm not sure if `IModelHost` should call `ECSchemaRpcImpl.register()` by default on startup. Personally I don't see why not, but that's why we should ask someone else :D @wgoehrig @aruniverse ?

Also, @GytisCepk would `ElectronRpcManager` always need to initialize an `ECSchemaRpcInterface` too?

Reply:
We are moving away from core registering rpcs by default. this will be handled by the meta framework / app that we are consolidating to on web and desktop

Reply:
> Also, @GytisCepk would ElectronRpcManager always need to initialize an ECSchemaRpcInterface too?

Yes. However, `ElectronRpcManager` is internal, so user would need to:
- Pass `ECSchemaRpcInterface` to `ElectronApp.startup`
- Pass `ECSchemaRpcInterface` to `ElectronHost.startup`
- Call `ECSchemaRpcImpl.register` in Electron backend

Regarding web, GPB currently already registers `ECSchemaRpcImpl`, so user only needs to register `ECSchemaRpcInterface` in the frontend

Reply:
Okay, I updated the doc comment for this method to provide a link to configuring RPC for client applications. I also added a snippet about configuring RPC for server-side, in cases of Electron backends, but maybe I should be clearer and point out Electron backends? Or just toss the snippet on server side stuff completely

Also added a note that BlankConnection's schemaContext will throw an error when trying to call any methods through the schemaContext.